tests/roots/test-markup-rubric/index.rst

```diff
@@ -5,3 +5,7 @@ test-markup-rubric
 
 .. rubric:: This is
    a multiline rubric
+
+.. rubric:: A rubric with a heading level
+   :level: 2
```

Comment:
Check levels 1 to 6 and invalid levels as well.

Reply:
done